app/assets/stylesheets/components/_photographer_carousel.scss

```diff
@@ -1,10 +1,10 @@
-.carousel.pointer-event {
-  border-style: solid;
-  border-width: 1px;
-  border-color: $gray;
-  padding: 10px;
-  border-radius: 10px;
-  line-height: 20px;
-  letter-spacing: 1px;
-  // min-width: 576px;
-}
+// .carousel.pointer-event {
```

Comment:
Why did you comment these out? They didnt have any effect?

Reply:
yes, it was in reference to the border dispute we talked about during the call.  Seems to work only randomly  @Matth93